libtalek/client.go

```diff
@@ -95,30 +96,42 @@ func (c *Client) MaxLength() uint64 {
 func (c *Client) Publish(handle *Topic, data []byte) error {
 	config := c.config.Load().(ClientConfig)
 
-	if len(data) > int(config.DataSize)-PublishingOverhead {
-		return errors.New("message too long")
-	} else if len(data) < int(config.DataSize)-PublishingOverhead {
+	if len(data) > 65535 {
```

Comment:
why 65535? If significant, should be a constant somewhere with a comment explaining